pkg/apis/extensions/types.go

```diff
@@ -82,43 +81,45 @@ type SubresourceReference struct {
 	Subresource string `json:"subresource,omitempty"`
 }
 
-// ResourceConsumption is an object for specifying average resource consumption of a particular resource.
-type ResourceConsumption struct {
-	Resource api.ResourceName  `json:"resource,omitempty"`
-	Quantity resource.Quantity `json:"quantity,omitempty"`
+type Utilization float64
```

Comment:
I think its impossible to utilize negative resources...

I don't see a major benefit in supporting a utilization target of 99.5% versus just 99%

Can this be an unint64?